compiled-lang/en.json

```diff
@@ -81,11 +81,12 @@
   "ruleIsDisabledForAllClusters": "Recommendation is disabled for all clusters",
   "ruleIsDisabledForAllSystems": "Recommendation is disabled for all systems",
   "ruleIsDisabledForClusters": "Recommendation is disabled for some clusters",
-  "ruleIsDisabledForClustersBody": "Recommendation is disabled for {clusters, plural, one {# cluster} other {# clusters}}",
+  "ruleIsDisabledForClustersBody": "Recommendation is disabled for {clusters, plural, one {# cluster} other {# clusters}} and has no results",
```

Comment:
A dot in the end of the sentence required. The same applies to other descriptors (I won't comment on them separately).